### ISSUE TITLE
Add AA insertion in nucleotide view tooltip: remove inconsistency

### ISSUE THREAD

**Issue by corneliusroemer** [nextstrain/nextclade#1105]
In nuc view, we show associated AA substitutions and deletions. However, we don't seem to show AA insertions (yet). This probably was forgotten when we added associated AA insertions.

Here AA could be shown:
<img width="714" alt="image" src="https://user-images.githubusercontent.com/25161793/218434206-e61a9031-4516-4e3f-98cd-67c132694697.png">

Similar to this:
<img width="549" alt="image" src="https://user-images.githubusercontent.com/25161793/218434240-c3b07f2f-6ca0-4558-b211-67955d2c79a3.png">
<img width="413" alt="image" src="https://user-images.githubusercontent.com/25161793/218434249-d1b60505-0bc1-4847-974e-355068e34c58.png">

**Comment by ivan-aksamentov** [comment]
@corneliusroemer Associating nucs and AAs is a tough task even for point mutations. Here you'd need to associate ranges. It wasn't forgotten, but it is not shown because this association is simply not implemented.

Do you have an algorithm in mind?

**Comment by corneliusroemer** [comment]
I see! Wouldn't it be very similar to deletions? I can see how you do it for deletions and try to port to insertions. Will put it on backlog

**Comment by ivan-aksamentov** [comment]
Good point. We handle deletions and insertions slightly differently, in that insertions are strippied away. But it might not matter.

I wonder about corner cases like: 
 - 1- or 2- nuc insertions - there is no full codon, so no AA isertion. Ir is there?
 - what if deletions and insertions are combined (adjacent, interspersed), i.e. indels in their most general case

**Comment by corneliusroemer** [comment]
Right, there may be corner cases, but in practice they will be rare. Associations are only done for the web ui so if there's an occasional odd sequence that should be ok.

Oh I see, it's the whole association that's missing - I first assumed it was just not shown.

1-2 nuc insertion is a frame shift. So we just don't show anything. If deletions and insertions are combined we show both - we already provide AAs for indels and nucs, just not linked to rough nuc position in case of insertions. I don't see a fundamental problem.

Actually, it would be cool to show insertions similarly to the way we show deletions:

No fundamental reason why that isn't possible:
<img width="964" alt="image" src="https://user-images.githubusercontent.com/25161793/218442015-5f391341-8e97-4fbc-9792-17d03e6491d4.png">

Rather than `---` at the bottom, you would have `---` at the top (or `+++` to make it less confusing?)
